src/main/java/mate/academy/spring/model/dto/response/OrderResponseDto.java

```diff
@@ -0,0 +1,44 @@
+package mate.academy.spring.model.dto.response;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import mate.academy.spring.model.User;
+
+public class OrderResponseDto {
+    private Long id;
+    private List<Long> ticketsIds;
+    private LocalDateTime orderDate;
+    private User user;
```

Comment:
```suggestion
public class OrderResponseDto {
    private Long id;
    private List<Long> ticketsIds;
    private LocalDateTime orderDate;
```
you are sending back user object with all the fields(password, salt, etc)
this dto is used in order controller where we get user id as input so no need to return it